src/ray/common/task/task_spec.h

```diff
@@ -142,6 +142,8 @@ class TaskSpecification : public MessageWrapper<rpc::TaskSpec> {
 
   bool IsDirectCall() const;
 
+  int MaxConcurrency() const;
```

Comment:
maybe `MaxActorConcurrency()` since this is for actors, and add a comment that this only applies to direct actor?